src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -108,6 +108,7 @@
   private View                        composeDivider;
   private View                        scrollToBottomButton;
   private TextView                    scrollDateHeader;
+  private int x                       =1;
```

Comment:
Is this being used anywhere? I can't find it.

Reply:
this has slipped